src/index.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Awesome books | Book App</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;1,100;1,200;1,300;1,400;1,500&display=swap" rel="stylesheet" />
+    <script src="https://kit.fontawesome.com/792c795fd6.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="./index.css" />
+  </head>
+
+  <body>
+    <header id="header" class="header">
+      <nav class="navBar">
+        <div class="mobile">
+          <a href="#" class="logo">Awesome Books</a>
+          <i class="fas fa-bars menubar"></i>
+        </div>
+        <ul class="nav-menu">
+          <li><a href="#" class="books blue">List</a></li>
+          <li><a href="#" class="create">Add new</a></li>
+          <li><a href="#" class="contact">Contact</a></li>
+        </ul>
+      </nav>
```

Comment:
- [ ] Clicking the navbar items doesn't do anything. Please note that it should render other pages. Kindly fix it.

Reply:
It works fine, the reason why it doesn't work is because the date throw an error

Reply:
- [ ] Your navbar is still not working kindly fix that as said by the first reviewer